.github/workflows/prepare-release.yaml

```diff
@@ -148,7 +158,7 @@ jobs:
           
           PR_URL=$(gh pr create --title "Release ${{ needs.bump-version.outputs.release-version }}" --body "## TODOs
           - [ ] Review the changes in [the release commit]($COMMIT_URL)
-          - [ ] Update the Release notes
+          - [ ] If major release: Update header of .pipeline/scripts/release_notes_template.md
```

Comment:
I'm not sure we'll need a dedicated reminder here that comes up with every minor release🤔

Reply:
Deleted the reminder.